R/openapi-types.R

```diff
@@ -70,6 +70,24 @@ add_api_info_onLoad <- function() {
     as.character,
     location = c("query", "path")
   )
+  addApiInfo(
+    "date-time",
+    c("POSIXct", "POSIXt"),
+    "[^/]+",
+    lubridate::as_datetime,
+    format = "date-time",
+    location = c("query", "path"),
+    realType = "string"
+  )
+  addApiInfo(
+    "date",
+    c("Date"),
```

Comment:
(Continuation)
```suggestion
    c("date"),
```

Reply:
I think this is my confusion here about what these parameters to `addApiInfo` mean... 

- are they openapi types? - in which case I guess `date` works
- or are they the R types? - in which case I think `Date` is better (`class(lubridate::today())`)

Currently the options I can see include lots of things which are openapi, lots which are R and some which are neither?

e.g. I can see all of:

```
    c("bool", "boolean", "logical"),
    c("dbl", "double", "float", "number", "numeric"),
    c("chr", "str", "character", "string"),
    c("POSIXct", "POSIXt"),
    c("list", "data.frame", "df", "object"),
```

I can see why we'd want to have more keys than are in the openapi spec - and I guess I'm ok with duplicates (e.g. dbl and double), but if there is a chance for a breaking change, then it might be nice to clean these up (especially **before** I add the very ugly POSIXct and POSIXt to the list 👍 )

Reply:
Resolving this as changes are included now